### PR TITLE
Day 18 develop

### PR DESCRIPTION
**Pull request by yujin0131**
Day 18 develop

### CODE REVIEW

File: src/main/java/com/bis/instagrarn/HomeController.java [hunk 1]
```diff
@@ -22,12 +28,28 @@
...
-	public String main(Locale locale, Model model) {
-
+	public String main(Model model, @RequestParam(value="page", defaultValue="0")int page) {
+		int user_idx = 1;
+		List<ProfileVO> list = profileService.select_post(user_idx, page);
+		model.addAttribute("list", list);
+		return Common.Board.VIEW_PATH + "main.jsp";
+	}
+	
+	@RequestMapping(value = "/loadpost", method = RequestMethod.GET)
```
Comment (yoonwooseong) [inline src/main/java/com/bis/instagrarn/HomeController.java:48]: Want to change the return value

File: src/main/webapp/WEB-INF/web.xml [hunk 2]
```diff
@@ -1,50 +1,51 @@
...
-	</servlet-mapping>
-
-</web-app>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app version="2.5" xmlns="http://java.sun.com/xml/ns/javaee"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://java.sun.com/xml/ns/javaee https://java.sun.com/xml/ns/javaee/web-app_2_5.xsd">
+	
+	<filter>
```
Comment (yoonwooseong) [inline src/main/webapp/WEB-INF/web.xml:6]: Need to remember this code to switch to utf-8

**Review by yoonwooseong — COMMENTED** [review]
Confirm

File: src/main/webapp/WEB-INF/views/profile/profile.jsp [hunk 3]
```diff
@@ -55,11 +68,39 @@
 				</div>
 			</div>
 		</div>
-	
-		<c:forEach var="list" items="${list}">
-			<img src="resources/post/${list.img}" width="200"/>
-		</c:forEach>
+		
+		<div class="menu_bar">
```
Comment (yoonwooseong) [inline src/main/webapp/WEB-INF/views/profile/profile.jsp:72]: Splitting it into two divs seems unnecessary

**Review by yoonwooseong — COMMENTED** [review]
Confirm

File: src/main/webapp/WEB-INF/views/profile/profile.jsp [hunk 4]
```diff
@@ -55,11 +68,39 @@
...
+				</div>
+			</div>
+
+		</div>
+		
+		
+      
+		<div class="user_post" id="user_post" style="width:972px;">
+			<c:forEach var="list" items="${list}">
+			<img src="resources/post/${list.img}" width="293" height="293" style="margin-right:27px;"/>
```
Comment (yoonwooseong) [inline src/main/webapp/WEB-INF/views/profile/profile.jsp:95]: Indentation is important :)

**Review by yoonwooseong — COMMENTED** [review]
Indentation is important

File: src/main/webapp/WEB-INF/views/header.jsp [hunk 5]
```diff
@@ -6,6 +6,25 @@
...
+	<script type="text/javascript">
+		function alert_new(){
+			
+			switch (document.getElementById("nav_alert_news").style.display) {
+			case "block":
+				document.getElementById("home_icon").src="${ pageContext.request.contextPath }/resources/images/IconHome2.png"
+				document.getElementById("heart_icon").src="${ pageContext.request.contextPath }/resources/images/IconHeart.png"
+				document.getElementById("nav_alert_news").style.display="none";
+				break;
+			case "none": case "":
```
Comment (yujin0131) [inline src/main/webapp/WEB-INF/views/header.jsp]: case "" : case "none": 
This order is more correct

**Pull request merged**